Description: enable building on freebsd
 The OSX code is generic enough to work, it just needed to be enabled.
 Tested on kfreebsd-amd64 and it works ok.
Author: Joey Hess <joeyh@debian.org>

---

--- haskell-network-info-0.2.0.1.orig/network-info.cabal
+++ haskell-network-info-0.2.0.1/network-info.cabal
@@ -40,7 +40,7 @@ Library
   build-depends:
     base == 4.*

-  if os(linux) || os(darwin)
+  if os(linux) || os(darwin) || os(freebsd)
     c-sources: cbits/network-unix.c
   else
     if os(windows)
